x-pack/plugins/infra/common/http_api/host_details/process_list.ts

```diff
@@ -84,7 +84,7 @@ const summaryPropertyRT = rt.union([rt.number, rt.string]);
 export const ProcessListAPIResponseRT = rt.type({
   processList: rt.array(
     rt.type({
-      cpu: rt.number,
+      cpu: rt.union([rt.null, rt.number]),
       memory: rt.number,
```

Comment:
While we haven't encountered problems with `memory` being null, do you think it's worth it to prevent it from causing crashes in this PR?

Reply:
Good point, I added similar logic for the memory and made the placeholder component reusable 👍